sdcm/cluster.py

```diff
@@ -2181,9 +2181,9 @@ def start_scylla(self, verify_up=True, verify_down=False, timeout=300):
         if verify_up:
             self.wait_jmx_up(timeout=timeout)
 
-    @retrying(n=3, sleep_time=5, allowed_exceptions=NETWORK_EXCEPTIONS,
-              message="Failed to stop scylla.server, retrying...")
-    def stop_scylla_server(self, verify_up=False, verify_down=True, timeout=300, ignore_status=False):
+    @retrying(n=3, sleep_time=5, allowed_exceptions=NETWORK_EXCEPTIONS + (CommandTimedOut, ),
+              message="Failed to stop scylla.server, retrying...", raise_on_exceeded=False)
+    def stop_scylla_server(self, verify_up=False, verify_down=True, timeout=3, ignore_status=False):
```

Comment:
why the timeout was changed?
why raise_on_exceeded=False? 
@roydahan did you mean that?

Reply:
> why the timeout was changed?

sorry, missed that. return back
> why raise_on_exceeded=False?

otherwise the exception will be raised that attempts exceeded and test will be stopped
> @roydahan did you mean that?

Reply:
I don't think the test will stop, would it?
If we raise the exception will it fail that nemesis?

Reply:
if stop failed, then nemesis failed, and next scylla start command will not be executed(in Drain monkey) and health validator will fail the job

Reply:
ok, but that's only if we couldn't stop the service after 3 retries, right?

Reply:
Yes

Reply:
in that case, I think it's ok to raise the exception so we can go and check it.

Reply:
if it raise an exception, then nemesis failed and node stay with stopped scylla and health validator detects it and stop the job

Reply:
I think we need to raise an exception and stop the test, otherwise, we will fail somewhere afterwards when for example trying to start Scylla server and when it wasn't stopped after 3 attempts and stuck in stopping state...

Reply:
Done